docs/output.md

```diff
@@ -2,58 +2,265 @@
 
 ## Introduction
 
-This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+This document describes the output produced by the pipeline based on public data.
 
-The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
+The directories listed below are created in the `results/` directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
+### Dataset
+
+RNAseq data has been taken from GM12878 GIAB sample SRA Accession [SRR5665260](https://www.ncbi.nlm.nih.gov/sra/?term=SRR5665260). The dataset has 38.6 million PE reads of read length 2x151 bp from NextSeq 500 sequencing platform.
+
+### Samplesheet
+
+A sample sheet has been prepared in the following way to set the FASTQ files to run the analysis.
+
+```console
+sample,fastq_1,fastq_2,strandedness
+GM12878,/data/GM12878/SRR5665260_1.fastq.gz,/data/GM12878/SRR5665260_2.fastq.gz,reverse
+```
+
+### Execution
+
+The pipeline has been executed with the following command.
+
+```console
+nextflow run nf-core/rnavar -profile munin,docker --input samplesheet.csv --genome GRCh38 --read_length 151 --annotate_tools merge
+```
+
+It used an institutional profile `munin` from [nf-core/configs](https://github.com/nf-core/configs/blob/master/conf/pipeline/rnavar/munin.config)
 
 ## Pipeline overview
 
-The pipeline is built using [Nextflow](https://www.nextflow.io/)
-and processes data using the following steps:
+The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
+
+- [Preprocessing](#preprocessing)
+  - [cat](#cat) - Merge re-sequenced FastQ files
+  - [FastQC](#fastqc) - Raw read QC
```

Comment:
There is not actually a FastQC section in the explanation below. Also I just noticed now that you some images in the folder which sound like they could be used for some more explanation with examples. I was just wondering why they are not included here

Reply:
Added some explanation of the results with additional QC output files and plots. QC section is in the documentation has been elaborated to show various type of QC metrics and multiQC reporting.